2022/_data/homepage.json

```
@@ -14,7 +14,7 @@
   "ticket_soon": "SOON",
   "ticket_soldout": "SOLD OUT",
   "ticket_staytuned" : "Stay tuned !!",
-  "tickets_url": "https://hopin.com/events/jbcnconf-2021-virtual/",
+  "tickets_url": "https://tickets.jbcnconf.com/event/2022",
```

The main URL should work (once we publish the event):
```suggestion
  "tickets_url": "https://tickets.jbcnconf.com/",
```